docs/flutter/v2/features/interruption-handling.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Interruption Handling
+nav: 3.27
+---
+
+You're chatting away on your video call and, uh-oh, a phone call comes in. No worries, the 100ms SDK abstracts away handling this situation for you.
+
+When a telephonic voice call comes in and the user accepts the following happens.
+
+-   All videos including your own are turned off.
+-   You are muted.
+
+When the call ends the SDK will restore all videos, taking care not to un-mute you if you had muted yourself before the call came in. Neither will it un-mute all the peers if you had muted them from the app.
+
+No code required, this happens automatically for all calls.
+
+In Android you will get an update on the `onRoomUpdate` method with the update type `HMSRoomUpdate.roomMuted` when the call start and `HMSRoomUpdate.roomUnmuted` when the call end.
```

Comment:
`OntrackUpdate` would be received by the app since the local audio and video tracks would be muted if an interruption happened

Reply:
ya we get `onTrackUpdate` as well adding it in doc.